manifests/d/Deskflow/Deskflow/1.17.0/Deskflow.Deskflow.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Deskflow.Deskflow
+PackageVersion: 1.17.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{0D79F5CB-9C83-4BCD-9500-19E369BE6303}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/deskflow/deskflow/releases/download/v1.17.0/deskflow_1.17.0+r1_windows_x64.msi

```

Comment:
@nicola02nb Please could you change this to:
https://github.com/deskflow/deskflow/releases/download/v1.17.0/deskflow-1.17.0.0_win64.msi

That way we don't need to have two MSI files, which causes some confusion.

Reply:
> @nicola02nb Please could you change this to: https://github.com/deskflow/deskflow/releases/download/v1.17.0/deskflow-1.17.0.0_win64.msi
> 
> That way we don't need to have two MSI files, which causes some confusion.

As far as I know someone will need to open a new PR for this change since this PR was already merged.

Reply:
Probably I can do it

Reply:
> Probably I can do it

Cool. I'm not sure if it's needed though. I asked a maintainer so you may want to hang fire for their response:
https://github.com/microsoft/winget-pkgs/pull/186364#issuecomment-2447109729

> As far as I know someone will need to open a new PR for this change since this PR was already merged.

Yes, that is what I was expecting would need to happen.

Reply:
Yeah, I'll wait, Probably when I'll PR it will ceck and verify the manifest another time...

Reply:
Opened PR #187073